tmail-backend/combined-identity/src/main/java/com/linagora/tmail/combined/identity/CombinedUsersRepositoryModule.java

```diff
@@ -42,6 +43,12 @@ public LdapRepositoryConfiguration provideConfiguration(ConfigurationProvider co
             configurationProvider.getConfiguration("usersrepository"));
     }
 
+    @Provides
+    @Singleton
+    public CassandraRepositoryConfiguration provideCassandraRepositoryConfiguration() {
+        return CassandraRepositoryConfiguration.DEFAULT;
```

Comment:
There was a missing Guice binding regarding the `CombinedUsersRepository`. Now with the password hash salting refactoring we need a binding to `CassandraRepositoryConfiguration` as our CombinedUsersRepository is using CassandraUsersDAO. But we don't really care about password hashing configuration in this case, as we just want to store the username, we never really use the password there. The test for login still occurs against ldap.

So I binded a dummy default one. Is that ok @chibenwa ?

Reply:
Looks great to me.